src/generators.rs

```diff
@@ -1072,6 +1072,173 @@ pub fn directed_binomial_tree_graph(
     })
 }
 
+/// Generate an undirected heavy hex graph. Fig. 2 of
+/// https://arxiv.org/abs/1907.09528
+/// An ASCII diagram of the graph is given by:
+///
+/// .. code-block:: console
+///     ... D-S-D   D ...
```

Comment:
This is a formatting error (and why ci is failing) because of the missing newline sphinx is interpretting the entire ascii art block as the type string for the codeblock. To fix this you can do:

```suggestion
/// .. code-block:: text
///
///     ... D-S-D   D ...
```

Reply:
Thanks!